docs/querying/post-aggregations.md

```diff
@@ -101,10 +101,13 @@ The expression post-aggregator is defined using a Druid [expression](math-expr.m
   "type": "expression",
   "name": <output_name>,
   "expression": <post-aggregation expression>,
-  "ordering" : <null (default), or "numericFirst">
+  "ordering": <null (default), or "numericFirst">,
+  "outputType": <output value type of expression> 
 }
 ```
 
+Output type is optional, and can be any native Druid type: `LONG`, `FLOAT`, `DOUBLE`, `STRING`, `ARRAY` types (e.g. `ARRAY<LONG>`), or `COMPLEX` types (e.g. `COMPLEX<json>`).
```

Comment:
This raises questions that the docs should answer:

- What benefit is there to providing `outputType`?
- What happens if `outputType` different from the type of `expression`? Error, cast, something else?